src/short-css-class-names/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+* Returns an array with valid CSS class names that are between 1 and 3 chars long.
+*
+* Return value in pseudo code:
+* [
+*    "A",
+*    "B",
+*    ...
+*    "Z__",
+* ]
+**/
+module.exports = function() {
+    "use strict";
+
+    let charRange = require("./char-range"),
+        digits = charRange("0", "9"),
+        letters = charRange("A", "Z").concat(charRange("a", "z")),
+        hyphenAndUnderscore = ["-", "_"];
+
+    let validFirstChars = letters,
+        allValidChars = digits.concat(letters).concat(hyphenAndUnderscore);
+
+    let classNames = [];
+
+    for (let firstChar of validFirstChars) {
```

Comment:
Why for...of? Isn't it just an array?
const classNames = validFirstChars.map(...) seems more suitable.